Ntuplizer/python/ntuplizerOptions_data_cfi.py

```diff
@@ -42,6 +42,6 @@
 config["DOMETRECLUSTERING"] = False
 
 #--------- JEC ----------#
-config["CORRJETSONTHEFLY"] = True
-config["CORRMETONTHEFLY"] = True
+config["CORRJETSONTHEFLY"] = False #JEC not available yet
```

Comment:
Applying no JEC to data will make the corrected softdrop mass rather unconsistent with MC corrected softdrop mass. Rather keep this True, and use the corrections from MC also for data (you changed this already above).